great_expectations_cloud/agent/message_service/asyncio_rabbit_mq_client.py

```diff
@@ -79,15 +83,19 @@ def stop(self) -> None:
         """Close the connection to RabbitMQ."""
         if self._connection is None:
             return
+        LOGGER.info("Stopping")
         if not self._closing:
             self._closing = True
-
-        if self._consuming:
-            self._stop_consuming()
-        self._connection.ioloop.stop()
+            if self._consuming:
+                self._stop_consuming()
+                self._connection.ioloop.run_forever()
```

Comment:
Wondering why we are calling run_forever in the stop()?

Reply:
https://github.com/great-expectations/cloud/pull/518#discussion_r1823407227